embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/config/behavior/OTelBehavior.kt

```diff
@@ -0,0 +1,32 @@
+package io.embrace.android.embracesdk.config.behavior
+
+import io.embrace.android.embracesdk.config.local.SessionLocalConfig
+import io.embrace.android.embracesdk.config.remote.RemoteConfig
+import io.embrace.android.embracesdk.internal.utils.Provider
+
+/**
+ * Provides the behavior that functionality relating to OTel should follow.
+ */
+internal class OTelBehavior(
+    thresholdCheck: BehaviorThresholdCheck,
+    remoteSupplier: Provider<RemoteConfig?>
+) : MergedConfigBehavior<SessionLocalConfig, RemoteConfig>(
+    thresholdCheck,
+    { null },
+    remoteSupplier
+) {
+    /**
+     * Returns whether OTel Stable is enabled.
+     */
+    fun isStableEnabled() = remote?.oTelConfig?.isStableEnabled ?: false
```

Comment:
The default for this should be true so we can ship some OTel features default on.

Reply:
Done